<launch>
    <node pkg="tf" type="static_transform_publisher" name="camera_fix" args="0 0 1.4 0.006017 0.4795145 0.0279046 0.8770695 zed_left_camera_frame camera_fixed 100" />
    <node pkg="liquid_height_estimation" name="height_detector" type="height_detector" output="screen" />
</launch>